lib/service.coffee

```diff
@@ -120,13 +120,18 @@ class Service
     this._execDatalinkLogs from, to, (err, logs) ->
       if err? then cb err # Continue with error.
       else # Log info.
+        skippedLogEntries = []
         logs.forEach (log) ->
-          if log.threshold?
-            console.log '[%s] %s %s - %s', log.threshold, chalk.green(log.containerId.substring(0, 12)), log.timestamp, chalk.cyan(log.message.trim())
+          if log?.message?
```

Comment:
Nested if statements can be hard to follow.  Try to avoid whenever you can.  One way would be:

```
unless log.message?
  log.skipped = true
  return skippedLogEntries.push log

if log.threshold?
   rest of for-each function goes here   
```